src/com/exadel/bootcamp/ClientAccount.java

```diff
@@ -8,11 +8,19 @@ public ClientAccount(String name) {
         this.name = name;
     }
 
-    public void put(CashMachine c, int cash) {
-        System.out.println("cash machine " + c + "user cash " + cash);
+    public void setAvailableCashAdd(int cash) {
```

Comment:
(naming) Just `addCash(int cash)`? Or even `add(int cash)`

Names are important for human readers, not for a compiler. Therefore, when you give a name, you must care about how it is being read.

1) No one likes wordiness. No one likes when you say it long what you could tell briefly.

2) People tend to read the whole line. Therefore, names of arguments are -- psychologically -- part of the name of the method. You don't need to write like `addLeftToRight(Left left, Right right)`. Because if you put it down like `add (Left left, Right right)` -- anyone will read it in his mind like "add left to right" - and that's what you wanted.